### ISSUE TITLE
[BUG] Another pause bug

### ISSUE THREAD

**Issue by Acros5542** [AoTTG-2/AoTTG-2#382]
**Describe the bug**
This one goes hand in hand with issue #381 as so far I've only found it to occur after #381 has occurred at least once since launching the game. After recreating #381, for any future lobbies you join or create, the first time you type /pause the game will give you the message in chat that the game has been paused, but it will not actually pause, nor will the "Game Paused" message appear in the center of your screen. Typing /pause a second time will actually pause the game, but the "Game Paused" message will still not appear. Typing /pause again will begin the countdown for the game to unpause, although this countdown will be invisible to you. For all future uses of /pause in the lobby, it will cycle between these two states as /pause would normally function; however, the "Game Paused" messages won't appear. In this specific lobby for singeplayer lobbies only, you can fix this issue by opening the menu with escape, after this the "Game Paused" messages will become visible; however, upon leaving the lobby and joining a new one, the issue will revert back to how it was before, with the first use of /pause not pausing the game and with the "Game Paused" messages being invisible. The only way to permanently fix this issue is to close the game entirely and relaunch.

**To Reproduce (including the steps for #381)**
1. Make a single or multiplayer session
2. Die to a titan or from impact if realism mode is on
3. Press 1 a single time
4. Type /restart before the game automatically restarts
5. Press escape and leave the lobby
6. Create a new singeplayer lobby and upon spawning in type /pause
7. Type /pause again to see the game actually pause
8. Type /pause a third time and wait a moment for the game to unpause
9. Press escape to open the menu, then return to the game
10. Type /pause again to notice the "Game Paused" message now appears
11. Clear the "Game Paused" message by typing /pause and wait for it to unpause
12. Leave the lobby and create a new one
13. Type /pause to notice the bug has reverted itself

**Expected behavior**
The first use of /pause on any lobby so long as your MC should pause the game and the "Game Paused" message should appear. The next use of /pause should begin the countdown to unpause the game. All future uses of /pause should cycle between these two states.

**Log file**
https://www.dropbox.com/s/54fo338osfuuusr/pause%20bug.txt?dl=0

**Additional context**
Considering this bug persists across lobbies and is caused by #381, I'd imagine #381 is doing something strange to your session overall beyond just the visible bug that occurs in the specific lobby you recreate #381 in.
Sorry this one is so wordy, I've got a video to go with it that I can upload early January.

**Contributor instructions**
1. Try to recreate the bug via the reported reproduction steps. If the bug can be reproduced, remove the investigation label. If the bug cannot be reproduced, please add a comment, attach "On Hold" label and inform the submitter that the bug cannot be reproduced. If there is no reply after 2 weeks, the issue can be closed.
2. Follow the default development team procedures (Create a new branch, apply your fix, create a pull request, submit new builds for testing team in dev squad)
3. Once testing team has confirmed the issue is resolved and the pull request is accepted, the issue can be closed.

**Comment by Acros5542** [comment]
Just encountered this bug again while testing another bug on a client that had not previously recreated #381. So it's a separate issue. I'm not entirely certain all of the ways you can cause this bug to occur, but I do know the steps listed above are a consistent way to make it happen.
